src/js/tabs/led_strip.js

```diff
@@ -1121,7 +1144,7 @@ led_strip.initialize = function (callback, scrollPosition) {
                 return mc.color;
             }
         }
-        return "";
+        return 3; //Throttle is the default
```

Comment:
3 what?

Reply:
3 is the index of the throttle in the list.

Reply:
Please be more specific, What list?

Reply:
![image](https://user-images.githubusercontent.com/62965528/189426004-79ca2bb6-f73c-4dee-a870-721f080e93f5.png)
This one